### PR TITLE
[TR] PIM-3994 Implement creating attribute options in the product form

### PR DESCRIPTION
**Pull request by filipsalpe**

### CODE REVIEW

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/attribute-option/create.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                                _.__('pim_enrich.form.attribute_option.flash.option_created')
+                            );
+                            deferred.resolve(option);
+                        }).fail(_.bind(function (xhr) {
+                            var response = xhr.responseJSON;
+
+                            if (response.children &&
+                                response.children.code &&
+                                response.children.code.errors &&
+                                response.children.code.errors.length > 0
```
Comment (Demango) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/attribute-option/create.js]: Wouldn't 0 equate to false anyway?

File: src/Pim/Bundle/EnrichBundle/Resources/public/js/attribute-option/create.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        messenger,
+        errorTemplate
+    ) {
+        var CreateOptionView = Backbone.View.extend({
+            errorTemplate: _.template(errorTemplate),
+            attribute: null,
+            initialize: function (options) {
+                this.attribute = options.attribute;
+
+                return BaseForm.prototype.initialize.apply(this, arguments);
```
Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/attribute-option/create.js]: Seems like an override of BaseForm, but Backbone.View is extended

Comment (jmleroux) [inline src/Pim/Bundle/EnrichBundle/Resources/public/js/attribute-option/create.js]: The constructor returns a BaseForm ?

**Comment by jmleroux** [comment]
GTM

**Pull request merged**